README.md

```diff
@@ -17,12 +17,20 @@ Software documentation is released under the Creative Commons license.
 * The custom [GeekDocs theme](https://geekdocs.de/) provides a bare-bones theme with templates and shortcodes for styling the website and content.
 * Specific .css customizations are added on top of GeekDocs in the /static/custom.css, /data/menu/, and /layouts/ files
 
-# Local Build Process
+# Installation and Download Process
 
 1. Download Hugo Extended v0.117.0 from [https://github.com/gohugoio/hugo/releases/](https://github.com/gohugoio/hugo/releases/tag/v0.117.0) and [install](https://gohugo.io/getting-started/installing/ "Install Hugo").
    See these instructions for manually installing the Hugo extended version in WSL: https://sal.as/post/install-hugo-on-wsl/
 
 2. Download this repository.
 
-3. Open the downloaded repository and run Hugo with `hugo serve`.
-   When complete, a local copy of the website is available by entering `localhost:1313` in a browser address bar.
+# Local Build Process
+
+1. Open the downloaded repository within your desired text editor and run Hugo with `hugo serve`. When complete, a local copy of the website is available by entering `//localhost:1313/` in a browser address bar.
+2. As changes within your chosen text editor or integrated development environment (IDE) are made, Hugo automatically rebuilds the local copy to include the intended modifications.
```

Comment:
Step 2 is not a step as there is no action, it is the result of running the command. Because this is just a one step/action instruction, remove the numbered list. 
Also, there was some incorrect information about what you enter the Hugo command in, it is entered not in a text editor, it is entered in a terminal window like PowerShell.  
Also, I've learned the hard way that the port 1313 is not always available so I made it a variable. All this in mind, I suggest the following change:
```suggestion
Open the downloaded repository in a terminal program like Windows Power Shell, then enter `hugo serve`.
When complete, a local copy of the website is available.
Enter <code>//localhost:<i>1313</i>/`</code> in a browser address bar, where <code><i>1313</i></code> is the port number shown in the text output of the Hugo build status. In most cases this is `1313` but can be a different port number.
Hugo automatically rebuilds the local copy to include any changes made and saved in your chosen text editor or integrated development environment (IDE).
```